src/consensus/tx_verify.cpp

```diff
@@ -74,18 +74,20 @@ bool CheckTransaction(const CTransaction& tx, bool fZerocoinActive, bool fReject
                     REJECT_INVALID, "bad-tx-version-too-high");
     }
 
+    // Size limits
+    BOOST_STATIC_ASSERT(MAX_BLOCK_SIZE_CURRENT >= MAX_TX_SIZE_AFTER_SAPLING);   // sanity
+    BOOST_STATIC_ASSERT(MAX_TX_SIZE_AFTER_SAPLING > MAX_ZEROCOIN_TX_SIZE);      // sanity
```

Comment:
nit: I know that this is a move-only but we could change it to use std::static_assert.

Reply:
yeah we can fix it directly here, since we are moving these lines.